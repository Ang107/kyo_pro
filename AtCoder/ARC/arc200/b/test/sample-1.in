3
4 3 5
1 1 7
8 6 11
