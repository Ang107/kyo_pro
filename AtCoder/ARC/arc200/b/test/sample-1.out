Yes
2025 200
No
Yes
20250615 200200
